docs/Using Fleet/enroll-hosts.md

```diff
@@ -289,7 +289,7 @@ expiration setting. To configure this setting, in the Fleet UI, head to **Settin
 
 Visit the Google Admin console. In the navigation menu, visit Devices > Chrome > Apps & Extensions > Users & browsers.
 
-Select the relevant organizational unit, users, or group where you want the fleetd Chrome extension to be installed.
+Currently, the Chrome extension can only be installed across the entire organization. The work to enable installation for sub-groups is tracked in https://github.com/fleetdm/fleet/issues/13353. 
```

Comment:
Just double-checking since these are the step-by-step instructions: does installing across the organization involve selecting any option on this screen, or is that the default if you don't select organizational units/groups/etc?

If there's no step required to select the entire org, I'd maybe pull this info into a blockquote since this is clarification about a limitation and not a step of the instructions.

Reply:
Agreed blockquote makes sense.

Reply:
```suggestion
Select the relevant organizational unit, users, or group where you want the fleetd Chrome extension to be installed.

> Currently, the Chrome extension can only be installed across the entire organization. The work to enable installation for sub-groups is tracked in https://github.com/fleetdm/fleet/issues/13353. 
```